### PR TITLE
OSDOCS-10982: updating external DNS tutorial to remove references to CDO

### PR DESCRIPTION
**Pull request by michaelryanmcneill**
Version(s): `enterprise-4.15`+

Issue: https://issues.redhat.com/browse/OSDOCS-10982

Link to docs preview: https://77817--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-external-dns.html

QE review: Not required for tutorials

Additional information:

### CODE REVIEW

File: cloud_experts_tutorials/cloud-experts-external-dns.adoc [hunk 1]
```diff
@@ -18,14 +18,7 @@ toc::[]
...
 
-[WARNING]
-====
-Starting with {product-title} 4.14, the Custom Domain Operator is deprecated. To manage Ingress in {product-title} 4.14, use the Ingress Operator. The functionality is unchanged for {product-title} 4.13 and earlier versions.
-====
-
-Configuring the xref:../applications/deployments/rosa-config-custom-domains-applications.adoc[Custom Domain Operator] requires a wildcard CNAME DNS record in your Amazon Route 53 hosted zone. If you do not want to use a wildcard record, you can use the `External DNS` Operator to create individual entries for routes.
-
-Use this tutorial to deploy and configure the `External DNS` Operator with a custom domain in {product-title} (ROSA).
+The External DNS Operator deploys and manages `ExternalDNS` to provide the name resolution for services and routes from the external DNS provider, like Amazon Route 53, to {product-title} (ROSA) clusters. In this tutorial, we will deploy and configure the External DNS Operator with a secondary ingress controller to manage DNS records in Amazon Route 53.
```
Comment (bhardesty) [inline cloud_experts_tutorials/cloud-experts-external-dns.adoc:21]: Can we be more specific about `ExternalDNS` here? For example, if it's a resource, you could say this: "...the `ExternalDNS` resource..."

File: cloud_experts_tutorials/cloud-experts-external-dns.adoc [hunk 2]
```diff
@@ -142,6 +150,40 @@ $ export ZONE_ID=$(aws route53 list-hosted-zones-by-name --output json \
   --dns-name "${DOMAIN}." --query 'HostedZones[0]'.Id --out text | sed 's/\/hostedzone\///')
 ----
 +
+. Prepare a document with the necessary DNS changes to enable DNS resolution for the canonical domain of the ingress controller:
```
Comment (bhardesty) [inline cloud_experts_tutorials/cloud-experts-external-dns.adoc:153]:
```suggestion
. Prepare a document with the necessary DNS changes to enable DNS resolution for the canonical domain of the Ingress Controller:
```

**Review by bhardesty — COMMENTED** [review]
Looks good! Just a couple tiny wording suggestions.

File: cloud_experts_tutorials/cloud-experts-external-dns.adoc (hunk 1 above)
Comment (michaelryanmcneill) [reply cloud_experts_tutorials/cloud-experts-external-dns.adoc:21]: as discussed, this isn't a resource but is an actual Kubernetes concept, so leaving this as is.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**